### PR TITLE
RUBBISH FOR COMPARISON IGNORE Wsbytebuffer 16439 2

### PR DESCRIPTION
**Pull request by hutchig**
OpenLiberty Pull Requester,

ATTENTION, READ THIS: Updated 4/11/2018 - Read and understand this completely,
then delete this entire template. If a reviewer or merger sees this template,
they should fail the review or merge.

If this code change is fixing a user-visible bug in previously released code, it MUST
have an associated issue mentioned in the PR text or description. That Issue also
MUST be labelled “release bug”

This directs automation to scrape this fix for inclusion in the next release's
list of bugs fixed.

If this code is NOT for fixing a released bug, for example new function, fixing
a bug in unreleased function, or other improvements that do not affect the
user-space, no label is needed. An issue could or could not be used based
on the developer’s discretion, but do still delete this text block.

For full details, please see this wiki page:

https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions

### CODE REVIEW

File: dev/com.ibm.ws.wsbytebuffer/resources/com/ibm/ws/bytebuffer/internal/resources/ByteBufferMessages.nlsprops [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+#COMPONENTNAMEFOR CWWKO Websphere kernel channel framework service: byte buffer support
+#CMVCPATHNAME com.ibm.ws.channelfw/resources/com/ibm/ws/bytebuffer/internal/resources/ByteBufferMessages.nlsprops
+#NLS_ENCODING=UNICODE
+#
+#ISMESSAGEFILE TRUE
+# NLS_MESSAGEFORMAT_VAR
+
+### WSBB will use message ids from 0600-0799
+
+NOT_VALID_CUSTOM_PROPERTY=CWWKO0600E: The custom property {0} has a value of {1}. This value is not valid.
```
Comment (helyarp) [inline dev/com.ibm.ws.wsbytebuffer/resources/com/ibm/ws/bytebuffer/internal/resources/ByteBufferMessages.nlsprops:26]: Please change `The custom property {0}` to `The {0} custom property` (use "the _variable_ noun" format). Applies to this message and to other messages in this file.

File: dev/com.ibm.ws.wsbytebuffer/resources/com/ibm/ws/bytebuffer/internal/resources/ByteBufferMessages.nlsprops [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+NOT_VALID_CUSTOM_PROPERTY.explanation=A custom property was specified with a value that is outside the range of valid values.
```
Comment (helyarp) [inline dev/com.ibm.ws.wsbytebuffer/resources/com/ibm/ws/bytebuffer/internal/resources/ByteBufferMessages.nlsprops:27]: Suggested change to the explanation:

A custom property value is outside the range of valid values.

File: dev/com.ibm.ws.wsbytebuffer/resources/com/ibm/ws/bytebuffer/internal/resources/ByteBufferMessages.nlsprops [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+NOT_VALID_CUSTOM_PROPERTY.useraction=Correct the configuration error.
+
+UNRECOGNIZED_CUSTOM_PROPERTY=CWWKO0601W: The custom property {0} specified for the WsByteBuffer component is not valid.
```
Comment (helyarp) [inline dev/com.ibm.ws.wsbytebuffer/resources/com/ibm/ws/bytebuffer/internal/resources/ByteBufferMessages.nlsprops:30]: Suggested change to the message:

The {0} custom property for the WsByteBuffer component is not valid.

File: dev/com.ibm.ws.wsbytebuffer/resources/com/ibm/ws/bytebuffer/internal/resources/ByteBufferMessages.nlsprops [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+UNRECOGNIZED_CUSTOM_PROPERTY.explanation=A custom property was specified incorrectly.
+UNRECOGNIZED_CUSTOM_PROPERTY.useraction=Correct the configuration error.
+
+CONFIG_VALUE_NUMBER_EXCEPTION=CWWKO0602E: The WsByteBuffer component generated a NumberFormatException exception when processing the following property: Property Name: {0}  Value: {1}
```
Comment (helyarp) [inline dev/com.ibm.ws.wsbytebuffer/resources/com/ibm/ws/bytebuffer/internal/resources/ByteBufferMessages.nlsprops:34]: Please change `NumberFormatException exception` to `NumberFormatException error` (replace "exception" with "error").

Unless the property and value names are long, maybe shorten `the following property: Property Name: {0} Value: {1}` to `the {0} property, which has a value of {1}`.

**Review by helyarp — COMMENTED** [review]
Comments have some suggested changes.

**Comment by skasund** [comment]
L2 message review completed

**Pull request closed without merging**